src/caret_analyze/plot/visualize_lib/bokeh/legend.py

```diff
@@ -251,9 +251,26 @@ def create_legends(
                 )
                 break
             legends.append(Legend(items=self._legend_items[i:i+10]))
-
         return legends
 
+    def create_legends_for_stacked_bar(
```

Comment:
`create_legends_bottom` may be appropriate. So that, we can use this function for another graph when needed.